pkg/agent/controller/networkpolicy/fqdn.go

```diff
@@ -636,22 +638,29 @@ func (f *fqdnController) parseDNSResponse(msg *dns.Msg) (string, map[string]ipWi
 	}
 	fqdn := strings.ToLower(msg.Question[0].Name)
 	responseIPs := map[string]ipWithExpiration{}
+	getMaxTTL := func(ttl1, ttl2 uint32) uint32 {
+		if ttl1 > ttl2 {
+			return ttl1
+		} else {
+			return ttl2
+		}
+	}
```

Comment:
there is a `max` built in in Golang: https://pkg.go.dev/builtin#max
It was introduced in Go 1.21